anonymizer/Makefile

```diff
@@ -1,4 +1,4 @@
-PLUGIN_VERSION=0.0.4
+PLUGIN_VERSION=0.0.5
```

Comment:
question (non blocking) : should we add a change log file? Could be useful for the plugin newsletter but yes not a must have

Reply:
As the plugin is deprecated, I don't think we should invest more time on it